modules/virtualization/microvm/common/dns-dhcp.nix

```diff
@@ -0,0 +1,32 @@
+# Copyright 2022-2023 TII (SSRC) and the Ghaf contributors
+# SPDX-License-Identifier: Apache-2.0
+{...}: let
+  domain = "ghaf";
+in {
+  # Dnsmasq is used as a DHCP/DNS server inside the NetVM
+  services.dnsmasq = {
+    enable = true;
+    resolveLocalQueries = true;
```

Comment:
A cosmetic change, but default for _[resolveLocalQueries](https://github.com/NixOS/nixpkgs/blob/nixos-23.11/nixos/modules/services/networking/dnsmasq.nix#L58)_ is true.

Reply:
Thanks, addressed in [301b72d](https://github.com/tiiuae/ghaf/pull/427/commits/301b72dfc704f6d22b27b177e418c0ce956589ca)